package.json

```diff
@@ -67,6 +67,7 @@
   },
   "dependencies": {
     "metacom": "^3.0.0-alpha.4",
+    "@oldendev/testplugin": "^1.1.6",
```

Comment:
Remove this

Reply:
This is only for testing, I referenced, that there must be cherry-pick only